### DISCUSSION TITLE
BLE Appears to have Brick ESP module

### DISCUSSION THREAD

**Discussion by Waz-Cpt** [revk/xESP32-Faikout#509] [General]
Hello There,

I tried enabling BLE to connect the XIAOMI BLE temp humidity sensor (flashed with the ATC firmware) and the device is no longer connecting to the wifi, nor is it bringing up the fallback AP. I also cannot connect the device to do physical debug or reload firmware.

Any advice to either reset settings to defaults via hardware or any other suggestions would be appreciated.

**Comment by revk** [comment]
Just refreshing over USB to be honest.

**Reply by Waz-Cpt** [reply]
The serial is blocked by something. Yes the USB to serial interface is working. But the serial to the SoC is blocked. Cannot fash firmware or even do physical serial debug

**Reply by revk** [reply]
I have no clue then, sorry. The ESP is connected to USB port directly.

**Reply by Waz-Cpt** [reply]
Yes USB connected directly.

**Reply by revk** [reply]
Sorry, just realised there may be some confusion. Do you have a picture of how you are connecting?

The board should have direct USB. I suggest using that rather than a serial lead, and seeing if that works.

**Reply by Waz-Cpt** [reply]
I am attempting to connect via the USB COM port of the board. I think that somehow enabling the BLE setting has tried to enable a pin or pins that were not meant to have been used and now the device is bricked.

Not sure if I can try using one of the other serial interfaces to debug the unit. Guess I need to just buy a new ESP module.

Can I just use an ESP32 WROOM or should I try find more S3 boards?

**Reply by revk** [reply]
Can you provide a picture - there should be 4 pins to allow USB directly which should work anyway.

**Reply by Waz-Cpt** [reply]
Using the Esp32 S3 devkit c

![IMG_3586](https://github.com/user-attachments/assets/7d21aadd-d69c-4105-a0ab-04d3e0ec6ab8)
![IMG_3585](https://github.com/user-attachments/assets/b19d088c-f9de-4471-a8b5-45f8bdbe96ef)

**Reply by revk** [reply]
Ok sorry I thought it was a faikin board. In which case, sorry, no clue.

**Reply by Waz-Cpt** [reply]
Is it poosible that enabling BLE did something with GPIO0 or GPIO46?

**Reply by revk** [reply]
I really see no way it could, sorry.
